recipes/nnpdf/recipe.yaml

```diff
@@ -0,0 +1,82 @@
+schema_version: 1
+
+context:
+  name: nnpdf
+  version: 4.0.10
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/NNPDF/nnpdf/archive/${{ version }}.tar.gz
+  sha256: 307bc48d51b44ab8cf24e365d1a89bbd50f62290f9a3142eae5c55188b9cb49d
+
+build:
+  number: 0
+  noarch: python
+  script:
+    - if: win
+      then:
+        - set POETRY_DYNAMIC_VERSIONING_BYPASS=${{ version }}
+      else:
```

Comment:
If this is noarch you don't need this noarch packages are built only on Linux.

Reply:
Ah, I see. I saw the error in the windows build and I tried to work around it. I'll remove it.